docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -550,8 +549,7 @@ A batch is a bundle of samples which is fed to a single iteration. If a batch is
 
 An epoch refers to a full, exhaustive visit to a dataset.
```

Comment:
**Line 550-552:** Proper indentation should be maintained under the bullet points.